src/components/ContextualSaveBar/README.md

```diff
@@ -91,6 +91,7 @@ Use the save action to provide an opportunity to save changes. Use the discard a
 <div style={{height: '250px'}}>
   <AppProvider
     theme={{
+      colors: {},
```

Comment:
These changes can be reverted now that we've split Theme and ThemeConfig, and colors remains an optional prop in ThemeConfig